project/src/components/favorites/favorites.tsx

```diff
@@ -1,4 +1,18 @@
-function Favorites(): JSX.Element {
+import {Offer} from '../../types/offer';
+import FavoritesOffersList from '../favorites-offers-list/favorites-offers-list';
+
+type FavoritesProps = {
+  offers: Offer[],
+}
+
+function Favorites({offers}: FavoritesProps): JSX.Element {
+  const favoriteLoc: Set<string> = new Set();
```

Comment:
Тут можно сразу в Set передавать filter от offers

```suggestion
  const favoriteLoc: Set<string> = new Set(offers.filter(offer => offer.isFavorite));
```